.travis.yml

```diff
@@ -93,7 +93,9 @@ before_script:
   - psql -c "create user lemur with password 'lemur;'" -U postgres
   - psql lemur -c "create extension IF NOT EXISTS pg_trgm;" -U postgres
   - npm config set registry https://registry.npmjs.org
-  - npm install -g npm@latest bower
+  - nvm install 16.13.0
```

Comment:
shall we add a comment to our future selves to update the version manually as needed?

Reply:
Done